internal/vulnerability/runner.go

```diff
@@ -70,3 +77,29 @@ func NewVulnerabilityRunner(log *logp.Logger) (VulnerabilityRunner, error) {
 func (f VulnerabilityRunner) GetRunner() artifact.Runner {
 	return f.runner
 }
+
+func clearTrivyCache(ctx context.Context, log *logp.Logger) error {
+	log.Info("Starting VulnerabilityRunner.ClearCache")
+	defer log.Info("Ending VulnerabilityRunner.ClearCache")
+
+	// These are the three available cli settings for clean/reset translated to flag.Options object.
+	// 	{CacheOptions: flag.CacheOptions{ClearCache: true}},
+	// 	{DBOptions: flag.DBOptions{Reset: true}},
+	// 	{MisconfOptions: flag.MisconfOptions{ResetPolicyBundle: true}},
+	// In our case we will use only the ClearCache option.
+
+	errs := make([]error, 0, 2)
+	r, err := artifact.NewRunner(ctx, flag.Options{CacheOptions: flag.CacheOptions{ClearCache: true}})
+	if err != nil {
+		if !errors.Is(err, artifact.SkipScan) {
+			errs = append(errs, err)
+		}
+	}
```

Comment:
nitpick: we don't need to wrap the with `if err != nil`, the `!errors.Is(err, artifcat.SkipScan)` already covers it (https://cs.opensource.google/go/go/+/master:src/errors/wrap.go;l=44)

```suggestion
	if !errors.Is(err, artifact.SkipScan) {
		errs = append(errs, err)
	}
```